### DISCUSSION TITLE
Créer des labels une seule fois pour tous les projets

### DISCUSSION THREAD

**Discussion by Amis-BASH** [community/community#147651] [Other Feature Feedback, Questions, & Ideas]
Question

Créer des labels une seule fois pour tous les projets.
Actuellement, les labels que j'ai modifié ne portent que sur un projet.

Create labels once for all projects.
Currently, the labels I have modified only relate to one project.

**Comment by shinybrightstar** [comment]
Hi @Amis-BASH , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.

**Comment by Amis-BASH** [comment]
` gh auth login`

```
#!/bin/bash

# Définir les labels à ajouter
labels=("bug" "enhancement" "documentation")

# List of repositories (Update this array with your repository names)
repos=("user/repo1" "user/repo2" "user/repo3")

for repo in "${repos[@]}"; do
  for label in "${labels[@]}"; do
    gh label create --repo $repo --name $label --color 000000
  done
done

```
Il faudrait un script plus adapté avec les labels existants, les enlever, et en créer de nouveaux pour les personnaliser.
